src/components/browse/components/table-commons/HeadersRow.js

```diff
@@ -120,6 +120,20 @@ export class HeadersRow extends React.PureComponent {
         return column.substring(0, column.length - 14);
     });
 
+    static getSortDirectionBySchemaFieldType(fieldType) {
+        switch (fieldType) {
+            case 'string':
+                return 'asc';
+            case 'integer':
+                return 'desc';
+            case 'number':
+                return 'desc';
+            case 'date':
+                return 'desc';
+        }
+        return null;
```

Comment:
It'd be slightly more performant (due to speed of dictionary/object compared to switch-case - ~ O(1) vs O(N)) to do something like this -

```suggestion
        const directionsByFieldType = {
            "string": "asc",
            "integer": "desc",
            "number": "desc",
            "date": "desc",
        };
        return directionsByFieldType[fieldType] || null;
```